Sources/Controllers/MyPlaces/OAFavoritesTabBarViewController.m

```diff
@@ -22,6 +23,15 @@ - (void)viewDidLoad
         [self setViewControllers:newTabs];
     }
     [self applyLocalization];
+
+    if (@available(iOS 15.0, *))
```

Comment:
This is not a correct fix for this issue. See https://developer.apple.com/forums/thread/682432